assets/src/components/vehicleRouteSummary.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { ComponentPropsWithoutRef, useContext } from "react"
+import { useRoute } from "../contexts/routesContext"
+import { StateDispatchContext } from "../contexts/stateDispatchContext"
+import vehicleLabel from "../helpers/vehicleLabel"
+import { emptyLadderDirectionsByRouteId } from "../models/ladderDirection"
+import { currentRouteTab } from "../models/routeTab"
+import { directionName, isVehicle } from "../models/vehicle"
+import { drawnStatus } from "../models/vehicleStatus"
+import { VehicleOrGhost } from "../realtime"
+import { RouteVariantName } from "./routeVariantName"
+import { ScheduleAdherence } from "./scheduleAdherence"
+import { Size, VehicleIcon, vehicleOrientation } from "./vehicleIcon"
+// import { VehicleProp } from "./vehiclePropertiesCard";
+interface VehicleOrGhostProp {
+  vehicle: VehicleOrGhost
+}
+
+// #endregion
+// #region Vehicle Summary
+const VehicleIcon1 = ({
```

Comment:
**question:** can this be renamed to something like `WrappedVehicleIcon` that's more descriptive than the `1` suffix?

Reply:
Let me know what you think of 2f0893bca125c73f272c90852a0779f3de1d4349 ?

Reply:
I like it!